docs/user-guide/widgets.md

```diff
@@ -57,10 +57,82 @@ Just below the chart's preview, the following configurations are available:
 
 !!! Note
     The *No operation* option is used when the aggregation method is not needed for the chart. If *No Operation* is selected, no aggregation will be carried out for the chart and the WFS service will be used to generate the chart without using the WPS process `gs:Aggregate` in GeoServer.
+    
+    
+* Enable the chart's legend by activating **Display Legend**
 
-* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*)
+* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*) or choose to **Customize the color**. 
 
-* Enable the chart's legend by activating **Display Legend**
+#### Color customization
+
+For *Bar Charts* and *Pie Charts*, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) provides the possibility to customize the colors of the charts bars and slices. From the **Color** option dropdown menu, the user can select the <img src="../img/button/custom-button.jpg" class="ms-docbutton"/> options and enter the **Custom Colors Settings** through the <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> button.
+
+<img src="../img/widgets/custom_colors_settings.jpg" class="ms-docimage"/> 
+
+Through this section, the user is allowed to:
+
+* Choose the **Color** of the bars or the slides through the *Color Picker*.
```

Comment:
* Change the default **Color** of bars or slices (depending on the chart type) through the *Color Picker*. This color will be applied for all values for which a *Class Color* has not been configured.